README.md

```diff
@@ -0,0 +1,17 @@
+# I Was Lost
+
+Pilot an overeager robot through confounding mazes.
+
+## Building
+
+```shell
+npm install
+```
+
+## Running
+
+```shell
+npm start
+```
+
+And then open this URL in your browser: http://localhost:8080/

```

Comment:
I have added a README file describing the project and including instructions on how to start the game.